src/components/sdk/MeetingControls/ContentShareControl.tsx

```diff
@@ -1,13 +1,14 @@
 // Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import React from 'react';
+import React, { useRef } from 'react';
 
 import { ControlBarButton } from '../../ui/ControlBar/ControlBarItem';
 import { ScreenShare } from '../../ui/icons';
 import { useContentShareState } from '../../../providers/ContentShareProvider';
 import { useContentShareControls } from '../../../providers/ContentShareProvider';
 import { PopOverItemProps } from '../../ui/PopOver/PopOverItem';
+import FileInput from '../../ui/Input/FileInput';
 
 interface Props {
   /** The label that will be shown for content share control, it defaults to `Content`. */
```

Comment:
@p-foucht  Do  you think we need a flag to allow customer to enable video file share in the dropdown list or not?

Reply:
Hmm, good question... It's not really a breaking change but an enhancement, so I think it might be fine to note require a flag

@davejlane Any chance you have an opinion on this kind of stuff?